newlib/libc/include/sys/features.h

```diff
@@ -39,7 +39,6 @@ extern "C" {
 /* Version with trailing underscores for BSD compatibility. */
 #define	__GNUC_PREREQ__(ma, mi)	__GNUC_PREREQ(ma, mi)
 
-
```

Comment:
This un-intentional white space change should not be included in this PR

Reply:
also done in 4a02c855a0a49f64a84412a39e88c41e7845c8ae